### ISSUE TITLE
Broken preview panel?

### ISSUE THREAD

**Issue by odie2** [inbasic/ignotifier#314]
Seems like preview panel broken for me with latest update - can't Trash, Mark as read or Spam (single message operations).

Using Opera Developer 30.0 and Gmail Notifier 0.5.4

**Comment by inbasic** [comment]
Do you get any error in the console?

**Comment by odie2** [comment]
No any errors in "Inspect element" of preview panel, settings page na currently page.

Also resigned in on Google and reinstalled extension. The same issue.

**Comment by inbasic** [comment]
I just checked the extension on my Opera. It works fine for me.

> No any errors in "Inspect element" of preview panel, settings page and currently page.

Okay then go to addon manager and click on the `Developer mode` button. Now open background page of the extension shown just below the extension. Now switch to the Console tab and see if you get an error there when you click on a button in the panel.

**Comment by odie2** [comment]
![image](https://cloud.githubusercontent.com/assets/9401878/7182627/ea6a5718-e452-11e4-9121-8d9a1e3e10b3.png)

**Comment by odie2** [comment]
Eventually if you want, I could give you TeamViewer access. I have no enought time for doing lists of extensions or screenshots these days.
If yes, let me know so I will write to you at Transifex.

**Comment by inbasic** [comment]
Sorry for the late reply. Seems like scheduling TeamViewer is not going to be easy due to our time differences. Anyhow, are you sure you haven't turn "Conversion mode" off in your Gmail?
https://support.google.com/mail/answer/5900?hl=en

The notifier currently only works if "Conversion mode" is on.

**Comment by odie2** [comment]
I have "Conversation view" enabled.

https://mail.google.com/mail/u/0/feed/atom returns unreaded message.

```<feed xmlns="http://purl.org/atom/ns#" version="0.3">
<title>Gmail - Inbox for maciejka45@gmail.com</title>
<tagline>New messages in your Gmail Inbox</tagline>
<fullcount>1</fullcount>
<link rel="alternate" href="https://mail.google.com/mail/u/2" type="text/html"/>
<modified>2015-04-20T19:29:20Z</modified>
<entry>
<title>
Re: [TB 2014 Chocianów] Tydzień 20.04. - 24.04. Podawajcie w komentarzach...
</title>
<summary>
Maksymilian Oberski skomentował(a) Twój post w na stronie TB 2014 Chocianów. Maksymilian Oberski
</summary>
<link rel="alternate" href="https://mail.google.com/mail/u/2?account_id=maciejka45@gmail.com&message_id=14cd8208c2ab7ce2&view=conv&extsrc=atom" type="text/html"/>
<modified>2015-04-20T18:39:13Z</modified>
<issued>2015-04-20T18:39:13Z</issued>
<id>tag:gmail.google.com,2004:1498997225121610978</id>
<author>
<name>Maksymilian Oberski</name>
<email>notification+kr4mbxemr4ma@facebookmail.com</email>
</author>
</entry>
</feed>```

Also no problem with fresh extension on my old, unused Google Chrome 42.0.2311.90


Could you provide me previous version, so I could check is that version bug?

**Comment by inbasic** [comment]
To get older versions go to the commit view,
https://github.com/inbasic/ignotifier/commits/multi-browser

then select a commit and download the project. There are some commits with `version ? is ready`, so you have access to the all versions there. Now you can point your Opera to install the project in the developer mode.

**Comment by odie2** [comment]
Now have got more bugs with latest version. And one of them looks like login problem.
I just recorded short video, so you could just pause if you will interested in any of detailed.

http://maciej.kuzniczysko.pl/files/notifier.avi

**Comment by inbasic** [comment]
The bug is definitively because of `XMLHttpRequest cannot ...`. If you click on the background.html link and go to the Console tab, you will see a more comprehensive bug report there.

Do you have Chrome too? Same error there as well?

**Comment by odie2** [comment]
On Chrome works fine.
Also I installed Chrome extension to Opera (by [Download Chrome Extension](https://addons.opera.com/pl/extensions/details/download-chrome-extension-9/)) and the same issue. Cannot do message's operations.

On background.html also only those "new" and "red".
![image](https://cloud.githubusercontent.com/assets/9401878/7520387/99e16436-f4e6-11e4-8c03-ba8645b0ee2f.png)

Will check that older version in a moment.

**Comment by odie2** [comment]
The same with dragging "src.safariextension" from 0.5.2

It have to be gmail-contact or browser bug.
Will install second instance of the same Opera if possible.

**Comment by odie2** [comment]
Eh, seems like something broken in my %appdata%.
When changed name to create fresh installation, extension working fine.
So have some work to migrate manually settings and similar from old installation or else check for deleting extensions rules etc. / disabling other extensions.

**Comment by odie2** [comment]
Did %appdata% copy, deleted Preferences and 2 Extensions [...] folders and 2 files, installed fresh installation (what's worked).
Deleted new %appdata%'s Opera Developer.

Revert old version and where send message to me (when was in trash and moved then to "Send") was working...

Also I missed that on extension page I cannot see any of installed extension.

**Comment by inbasic** [comment]
I still don't see how it could be your profile problem, as only those panel operations were not working. Seems like some sort of https access problem. Anyhow it is good to know the new profile could have solve such a problem

I found this regarding moving to a new profile. Is this helping?
http://superuser.com/questions/367741/how-do-i-transfer-my-google-chrome-profile-to-a-new-pc

**Comment by odie2** [comment]
Seems like automatically fixed with (I think) 5.5 update ;)
